app/assets/javascripts/miq_application.js

```diff
@@ -89,36 +89,19 @@ function miqInitWidgetPulldown() {
   var miqMenu = new dhtmlXToolbarObject("widget_select_div", "miq_blue");
   miqMenu.setIconsPath("/images/icons/24/");
   miqMenu.attachEvent("onClick", miqWidgetToolbarClick);
-  // miqMenuXML var is loaded in dashboard/_dropdownbar.rhtml
-  miqMenu.loadXMLString(miqMenuXML);
+  // ManageIQ.widget.menuXml var is loaded in dashboard/_dropdownbar.rhtml
+  miqMenu.loadXMLString(ManageIQ.widget.menuXml);
   miqSetToolbarButtonIds(miqMenu);
 }
 
 function miqCalendarDateConversion(server_offset) {
-  var date = new Date();
-  var x = new Date(
-    date.getUTCFullYear(),
-    date.getUTCMonth(),
-    date.getUTCDate(),
-    date.getUTCHours(),
-    date.getUTCMinutes(),
-    date.getUTCSeconds()
-  );
-  if (server_offset >= 0) {
-    x.setTime(x.getTime() + (Number(server_offset)) * 1000);
-  } else {
-    x.setTime(x.getTime() - (Number(server_offset.slice(1, server_offset.length))) * 1000);
-  }
-  return x;
+  return moment().utcOffset(Number(server_offset) / 60);
 }
 
 // Track the mouse coordinates for popup menus
-var miqMouseX;
-var miqMouseY;
-
 function miqGetMouseXY(positionX, positionY) {
```

Comment:
@romanblanco this function is called only at one place and it's really simple so you can copy its body to the [place](https://github.com/ManageIQ/manageiq/blob/d7f73b53b01d1243f9e97df325fc5c252f7e5ce7/app/assets/javascripts/miq_application.js#L22) where it is called